hibernate-core/src/main/java/org/hibernate/collection/internal/PersistentBag.java

```diff
@@ -34,8 +34,12 @@
  */
 public class PersistentBag extends AbstractPersistentCollection implements List {
 
+	// TODO: Why is this.bag protected? Can it be changed to private?
```

Comment:
As we've discussed elsewhere this is a historical pattern for inheritance.  Feel free to change these to use protected getter/setter if you prefer.